### PR TITLE
Add simple NFT scaffold

### PR DESCRIPTION
**Pull request by nvdtf**
Closes https://github.com/onflow/flow-cli/issues/909

## Description

Add new scaffold for simple Cadence NFT project: https://github.com/nvdtf/flow-nft-scaffold
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: scaffolds.json [hunk 1]
```diff
@@ -11,6 +11,12 @@
     "description": "Scaffold contains required folder structure as well as some example Cadence code.",
     "commit": "f14d43b951400b95e850c07023735538ff836ac6"
   },
+  {
+    "name": "Cadence NFT Project",
+    "repo": "https://github.com/nvdtf/flow-nft-scaffold.git",
+    "description": "Scaffold contains the ExampleNFT sample NFT contract.",
+    "commit": "624ffd71bd2f86faf954d251f69d346ddff1446a"
```
Comment (sideninja) [inline scaffolds.json:18]: does it make sense to be categorized under any of these types: web, mobile, unity?

Comment (nvdtf) [reply scaffolds.json:18]: none of web, mobile, unity apply to the scaffold, it's just cadence code importing NonFungibleToken

**Comment by nvdtf** [comment]
@sideninja why the security scanner is failing here?

**Comment by sideninja** [comment]
> @sideninja why the security scanner is failing here?

Unralated issue. Will fix.

**Review by sideninja — APPROVED** [review]

**Pull request merged**